Frameworks/UIKit/UILabel.mm

```diff
@@ -576,13 +547,26 @@ - (UILineBreakMode)lineBreakMode {
  @Status Interoperable
 */
 - (void)setHighlighted:(BOOL)highlighted {
-    _isHighlighted = highlighted;
-    if (highlighted) {
-        [self setBackgroundColor:nil];
-    } else {
-        [self setBackgroundColor:_savedBackgroundColor];
+    if (highlighted != _isHighlighted) {
+        _isHighlighted = highlighted;
+        if (_isHighlighted) {
+            [self setBackgroundColor:nil];
```

Comment:
Yeah. it maybe some special behaviour for TextBlock. I can take a look at this separately since this is existing code.